src/nvim/shada.c

```diff
@@ -2318,6 +2318,55 @@ static inline ShaDaWriteResult shada_read_when_writing(
   return ret;
 }
 
+/// Get list of buffers to write to the shada file
+///
+/// @param[in] khash_t(bufset)   Buffers which are ignored
+///
+/// @return ShadaEntry  kSDItemBufferList of buffers to save
+static ShadaEntry shada_get_buflist(khash_t(bufset) * const removable_bufs)
```

Comment:
FUNC_ATTR_NONNULL_ALL FUNC_ATTR_WARN_UNUSED_RESULT

Also in other places it is written without space between `*` and `const`.